packages/reports/src/Components/Form/CompositeCalendar/side-list.tsx

```diff
@@ -1,16 +1,28 @@
-import PropTypes from 'prop-types';
 import React from 'react';
 import { toMoment } from '@deriv/shared';
 import ListItem from './list-item.jsx';
 
-const isActive = (from, to, flag) => {
+type TItem = {
+    value: string;
+    label: string;
+    onClick: () => void;
+    duration: number;
```

Comment:
```suggestion
type TItem = {
    value: string;
    label: string;
    onClick: () => void;
    duration: number;
```

@aizad-deriv, somehow, I think this type needs to sync with the one for `ListItem`, you might get errors when the 2 PRs are merged together.